test/cases/entry-exports-field/imports/index.js

```diff
@@ -0,0 +1,6 @@
+import d1 from "pck";
```

Comment:
`Cannot find module 'pck'`

The `#internal` seem to work

Reply:
I've simplified the test case, the error does persist.